### ISSUE TITLE
Deprecations and semantic issues

### ISSUE THREAD

**Issue by tyronedougherty** [floatinghotpot/cordova-iad-pro#2]
There are a number of deprecations and semantic issues that throw warnings in Xcode that need to be fixed.
![Here's a screenshot of the warnings in Xcode](https://cloud.githubusercontent.com/assets/4608024/6317362/0ef95406-baa2-11e4-82d9-acd3be8ad9f6.png)

**Comment by rubberbird** [comment]
Yes, I noticed this too. If you could sort these out it would be great.
